dmcontent/questions.py

```diff
@@ -626,6 +662,20 @@ def answer_required(self):
             return self.is_empty
 
 
+class DateSummary(QuestionSummary):
+
+    def __init__(self, question, service_data):
+        super(DateSummary, self).__init__(question, service_data)
+        self._value = self._service_data.get(self.id, '')
+
+    @property
+    def value(self):
+        try:
+            return datetime.strptime(self._value, '%Y-%m-%d').strftime('%A %-d %B %Y')
```

Comment:
If we are formatting it into the common digital marketplace format we would ideally use the dmutils `DATE_FORMAT` instead of duplicating